packages/components/src/components/link/link.component.ts

```diff
@@ -63,11 +62,49 @@ class Link extends DisabledMixin(IconNameMixin(Component)) {
   size: LinkSize = DEFAULTS.LINK_SIZE;
 
   /**
-   * Used to store the previous tabindex value of the host element
-   * null value means that the host element did not have a tabindex attribute.
+   * Href for navigation. The URL that the hyperlink points to
+   */
+  @property({ type: String, reflect: true })
+  href: string = '#';
+
+  /**
+   * Optional target: _blank, _self, _parent, _top and _unfencedTop
+   */
+  @property({ type: String, reflect: true })
+  target: string = '_self';
```

Comment:
```suggestion
  @property({ type: String, reflect: true })
  href = '#';

  /**
   * Optional target: _blank, _self, _parent, _top and _unfencedTop
   */
  @property({ type: String, reflect: true })
  target = '_self';
```

When a variable is declared with a default value, then you don't need to provide the type.

Reply:
Okay, I see. But we still add the typeInterface sometimes. Let's take the size attribute as an example — should we remove it here as well, since we're already providing a default value for it?
```
  /**
   * Size of the link.
   * Acceptable values include:
   *
   * - 'small'
   * - 'midsize'
   * - 'large'
   *
   * @default large
   */
  @property({ type: String, reflect: true })
  size: LinkSize = DEFAULTS.LINK_SIZE;
```

Reply:
For any property which does not have the basic types, you do need to define them.